### PR TITLE
[kubernetes] Update version

### PR DESCRIPTION
**Pull request by ahmedyarub**
**Describe the pull request**

- #### What does your PR fix?
  pdates Kubernetes Client to the latest version

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/kubernetes/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "kubernetes",
-  "version-date": "2022-01-19",
-  "port-version": 1,
+  "version-date": "2022-11-19",
+  "port-version": 2,
```
Comment (Cheney-W) [inline ports/kubernetes/vcpkg.json:4]:
```suggestion
```
If you update the `version-date`, the `port-version` should be remove.

File: ports/kubernetes/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   "description": "Kubernetes C client",
   "homepage": "https://github.com/kubernetes-client/c/",
```
Comment (Cheney-W) [inline ports/kubernetes/vcpkg.json:6]:
```suggestion
  "homepage": "https://github.com/kubernetes-client/c/",
  "license": "Apache-2.0",
```

File: ports/kubernetes/vcpkg.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "kubernetes",
-  "version-date": "2022-01-19",
-  "port-version": 1,
+  "version-date": "2022-11-19",
```
Comment (Cheney-W) [inline ports/kubernetes/vcpkg.json:3]: 2022-11-19
How did you get this date? I can't find the information related to this date, and the date of the latest commit merge is not it.

Comment (ahmedyarub) [reply ports/kubernetes/vcpkg.json]: Fixed.

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**